src/Bicep.Local.Extension/Host/Attributes/BicepTypeAttribute.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Bicep.Local.Extension.Host.Attributes;
```

Comment:
For namespaces what about `Bicep.Local.Extension.Types.Attributes` (and `Bicep.Local.Extension.Types.TypeBuilder`)?